gpytorch/lazy/lazy_variable.py

```diff
@@ -818,13 +818,18 @@ def zero_mean_mvn_samples(self, n_samples):
 
     def __add__(self, other):
         from .sum_lazy_variable import SumLazyVariable
-
+        if other is None:
+            return self
         return SumLazyVariable(self, other)
 
     def __div__(self, other):
+        if other is None:
+            return self
         return self.mul(1. / other)
 
     def __mul__(self, other):
+        if other is None:
+            return self
```

Comment:
should we raise an error when multiplying / dividing with what essentially is a zero kernel?